aria/data/midi.py

```diff
@@ -114,8 +113,11 @@ def __init__(
                 }
             ]
 
+    @classmethod
+    @property
+    def program_to_instrument(cls):
```

Comment:
I'm assuming this is here to speed up the process of building MidiDatasets. Does it make much of a difference?